network-api/networkapi/templates/pages/base.html

```diff
@@ -110,8 +110,8 @@
         {% endif %}
 
         {% block donate_banner %}
-            {% if settings.donate_banner.SiteDonateBanner.active_donate_banner %}
-                {% include "fragments/donate_banner.html" with banner=settings.donate_banner.SiteDonateBanner.active_donate_banner.localized %}
+            {% if donate_banner %}
+                {% include "fragments/donate_banner.html" with banner=donate_banner.localized %}
```

Comment:
Just a note: I only see a banner on `en` locale

Reply:
@robdivincenzo it happend to me as well - I just synced Homepage with translated pages and worked fine 👍

Reply:
Thanks for the heads up, @robdivincenzo, and thanks for reviewing, @dlopezvsr!

@robdivincenzo, as Diego mentioned, the `donate_banner` field is currently a synchronized field. This means that when we update it on the English page, we have to sync the translations for the change to be reflected across other locales.

This also brings up an important point for discussion: "A/B testing on other locales."

At the moment, Wagtail’s A/B testing functionality only runs the test on the English version of the page out of the box. As a result, visitors to other locales are treated as if no A/B test is active and are not included in the test—they are simply served the control variant.

However, we could modify our custom method to serve a variant cookie from the A/B test, and still serve the localized version of the page/donate banner. 

I've reached out to Genevieve to confirm if the Engagement team would like the A/B test extended to all locales. I'll update this thread once I hear back.

In the meantime, I was wondering if you had any feedback on this suggested approach?

Thanks!